source/server/index.js

```diff
@@ -1,5 +1,11 @@
 import createCuid from '../index.js';
-const fingerprint = require('node-fingerprint')();
+
+// workaround for webpack's process polyfill
+try {
+  process.pid = process.pid || Date.now();
```

Comment:
Yeah, that's what I initially thought, but I wasn't sure which would be the better solution. In hindsight assigning it to a new var does seem like it should be safer.